nav2_motion_primitives/src/stop.cpp

```diff
@@ -18,14 +18,13 @@
 
 #include "nav2_motion_primitives/stop.hpp"
 
-using nav2_tasks::TaskStatus;
 using namespace std::chrono_literals;
 
 namespace nav2_motion_primitives
 {
 
 Stop::Stop(rclcpp::Node::SharedPtr & node)
-: MotionPrimitive<nav2_tasks::StopCommand, nav2_tasks::StopResult>(node)
+: MotionPrimitive<StopAction>(node, "Stop")
```

Comment:
Looking at these action files: https://github.com/ros-planning/navigation2/blob/master/nav2_tasks/include/nav2_tasks/stop_action.hpp

They're all less than 5 lines of meaningful code. Also they're not Tasks anymore, they probably don't belong in `nav2_task` package. I think that this is a great time to merge the action into the `motion_primitives` package and/or directly removing that separate class

Reply:
> I dont think you should stop here because this is inside of motion_primitives. Its more than possible there's a primitive where that is nonsensical or potentially detrimental. I wouldn't assume how people will use this in their applications.

Will remove `stopRobot` as mentioned above.

Reply:
Yes, already done on the `lifecycle` branch.